src/app/PCM/templates/vert-and-hori/hori.js

```diff
@@ -12,16 +13,18 @@ const hori = mappingFunction => (mappedPcm, depth, config) => {
   const isPCMEvents = equals(pcmName, 'PCM Events')
   const isFormsPcm = includes('_FORM', pcmCode)
 
+  const isInternmatch = useIsProductInternmatch()
+
   return (
     <Stack
       spacing={isMobile ? 1 : 5}
       direction={isMobile && !isPCMEvents ? 'column' : 'row'}
       w={'min(100%, 50rem)'}
       justifySelf={'flex-start'}
-      justify={tplHoriJustify}
-      alignItems={isFormsPcm ? 'flex-end' : 'flex-start'}
+      justifyItems={tplHoriJustify}
+      alignItems={isFormsPcm ? 'flex-start' : 'flex-start'}
```

Comment:
Can you make this flex-end on IM forms? The fields seems unaligned if they are made flex-start on focus.

Reply:
ah that was a mistake I left in while testing, I will fix that